### PR TITLE
Generate initial cleaned plant cover data

### PR DESCRIPTION
**Pull request by KristinaRiemer**
As described in [the README](https://github.com/genophenoenvo/neon-datasets/compare/main...KristinaRiemer:curate_cover?expand=1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5), `curate_data.R` downloads the "Plant presence and percent cover" NEON data using the `neonstore` package, cleans the data, and produces the `plant_cover.csv`. This is currently for a single site and month of data, but can very easily be expanded to additional sites and time ranges.

### CODE REVIEW

File: plant_cover/curate_data.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                                 start_date = "2019-04-01", end_date = "2019-04-30")
+
+curated_df <- single_site_date_df %>% 
+  select(uid, siteID, decimalLatitude, decimalLongitude,
+         plotID, subplotID, endDate, scientificName, taxonRank,
+         percentCover) %>% 
+  filter(!is.na(scientificName), 
+         taxonRank == "species") %>% 
+  select(-taxonRank) %>% 
+  mutate(decimalLatitude = round(decimalLatitude, 2),
```
Comment (dlebauer) [inline plant_cover/curate_data.R:26]: Maybe 4 decimal places?

image

File: README.md [hunk 2]
```diff
@@ -1,2 +1,107 @@
...
+## Curated NEON datasets
+
+Contains scripts for downloading and cleaning data, and the resulting data files. 
+Metadata for original and curated datasets are in this README. 
+
+### 1. Plant cover
+
+#### Original data
+
+- **Plant presence and percent cover** dataset
```
Comment (dlebauer) [inline README.md:10]: some useful information here, but not sure if the columns of the raw data are required.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,107 @@
...
+  - Scripts
+    - `curate_data.R` cleans up data
+    - `summarize_data.R` creates summary stats, tables, and figures of data
+  - Derived data and figures
+    - `plant_cover.csv` is curated data
+    - All `.png` images result from `summarize_data.R` and are displayed below
+
+#### Curated data details
+
+Columns:
```
Comment (dlebauer) [inline README.md:70]: each column name should have a description

**Review by dlebauer — APPROVED** [review]
Looks great - out of curiosity, is there a reason not to use a README.Rmd, and generate the tables as text rather than as images?

File: README.md (hunk 2 above)
Comment (KristinaRiemer) [reply README.md:10]: Removed those [here](https://github.com/genophenoenvo/neon-datasets/pull/1/commits/71ca2972549807254bd4fd1c155ba0468b49c654).

File: README.md (hunk 3 above)
Comment (KristinaRiemer) [reply README.md:30]: Added descriptions [here](https://github.com/genophenoenvo/neon-datasets/pull/1/commits/71ca2972549807254bd4fd1c155ba0468b49c654).

**Comment by KristinaRiemer** [comment]
@dlebauer no particular reason except that it didn't occur to me until partway through doing it this way. It would be much easier that way though, so there's [an issue](https://github.com/genophenoenvo/neon-datasets/issues/4) for doing so.

**Pull request merged**